### PR TITLE
ntru-prime

### PR DESCRIPTION
**Pull request by franziskuskiefer**
some more data on ntru-prime

### CODE REVIEW

File: encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+platform: xeon-e3-1220-skylake
+impl: factored
```
Comment (benj-zen) [inline encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml]: There is a mismatch between the filename and this attribute.

File: encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+param: '653'
```
Comment (benj-zen) [inline encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml]: Also a mismatch 653 != ntrulpr653

File: encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml (hunk 1 above)
Comment (franziskuskiefer) [reply encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml]: If that's something we rely on, it should be checked by the validator script.

Comment (benj-zen) [reply encryption/ntru-prime/lprime/bench/optimized_ntrulpr653_xeon-e3-1220-skylake.yaml]: PR #23 addresses that.

**Pull request merged**